tests/functional-swap.rs

```diff
@@ -415,8 +411,8 @@ fn retry_until_monero_funding_address(args: Vec<String>) -> monero::Address {
         let monero_addresses: Vec<String> = stdout
             .iter()
             .filter_map(|element| {
-                if element.contains("Success: 4") {
-                    let monero_address = element[9..].to_string();
+                if let Some(pos) = element.find("XMR to 4") {
+                    let monero_address = element[pos + 7..].to_string();
```

Comment:
this is insanely hard to maintain

Reply:
I agree, the parsing of the output is super ugly right now, will improve this obviously.